boards/arm/lpcxpresso55s36/pinmux.c

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2023 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/init.h>
+#include <zephyr/drivers/pinctrl.h>
+#include "fsl_common.h"
+
+static int lpcxpresso55s36_pinmux_init(void)
+{
+	int err;
+
+	/* Declare pin configuration state for USB pin here */
+	PINCTRL_DT_DEFINE(DT_NODELABEL(usbfs));
+
+	/* Apply pinctrl state directly, since there is no flexio device driver */
+	err = pinctrl_apply_state(PINCTRL_DT_DEV_CONFIG_GET(DT_NODELABEL(usbfs)),
```

Comment:
why does this not work from DT? there needs to be a flexio driver?

Reply:
FlexIO is not related to this, so I think that is an error in the comment

Reply:
This code has been deleted,